api/server/router/build/build_routes.go

```diff
@@ -72,17 +70,7 @@ func newImageBuildOptions(ctx context.Context, r *http.Request) (*types.ImageBui
 	options.Target = r.FormValue("target")
 	options.RemoteContext = r.FormValue("remote")
 	if versions.GreaterThanOrEqualTo(version, "1.32") {
-		apiPlatform := r.FormValue("platform")
-		if apiPlatform != "" {
-			sp, err := platforms.Parse(apiPlatform)
```

Comment:
should we keep the validation/error-handling as it was before we changed to this? https://github.com/moby/moby/pull/37350/files#diff-f0ccc740f5a5ed38075be6855298809cL77

Reply:
Oh, I see, you moved the validation to the build/next-builder.